landavailability/tests/lr/test_models.py

```diff
@@ -81,8 +81,9 @@ def test_uprn_model_creation(self):
         self.assertEqual(Polygon.objects.count(), 1)
 
         uprn = Uprn()
-        uprn.title = title
         uprn.uprn = "0031535421432"
+        uprn.save()  # we need an id before we can add a M2M relationship
+        uprn.titles.add(title)
```

Comment:
This definitely tests if we can associate a Title to a Uprn. We should also include a test to verify the opposite thing: if we can add a Uprn to a Title.